Resources/Prototypes/Roles/Jobs/Civilian/chaplain.yml

```diff
@@ -22,6 +22,7 @@
     components:
     - type: PsionicBonusChance #Nyano - Summary: makes it more likely to become psionic.
       multiplier: 3
+    - type: Psionic
```

Comment:
This seems to have both a Psionic component _and_ a PsionicBonusChance--is the BonusChance still necessary when the Psionic component is added?

Reply:
It's definitely not, since any entity that has the PsionicBonusChance is going to require you have the PsionicComponent in the first place, it makes sense to merge whatever datafields PsionicBonusChance has with PsionicComponent, and maybe add an Event that allows arbitrarily any system to modify bonus chances.